src/BodyTable.js

```diff
@@ -13,12 +14,21 @@ export default function BodyTable(props, { table }) {
     handleBodyScroll,
     handleWheel,
     expander,
+    rowWidth,
     isAnyColumnsFixed,
   } = props;
   const { saveRef } = table;
   let { useFixedHeader } = table.props;
+  const { virtualized } = table.props;
   const bodyStyle = { ...table.props.bodyStyle };
   const innerBodyStyle = {};
+  if (virtualized) {
+    bodyStyle.width = rowWidth;
+    bodyStyle.height = virtualized.bodyHeight;
```

Comment:
Does `virtualized.bodyHeight` conflict with `scroll.y`?

Reply:
Seriously, there is no reason for one to use scroll.y and virtualized at the same time.
However, to be more precise, we can write some thing in the doc telling the user that these two features cannot co-exist.